android/build_boinc_arm.sh

```diff
@@ -15,25 +15,23 @@ VERBOSE="${VERBOSE:-no}"
 
 export BOINC=".." #BOINC source code
 
+export NDK_ROOT=${NDK_ROOT:-$HOME/Android/Ndk}
 export ANDROID_TC="${ANDROID_TC:-$HOME/android-tc}"
 export ANDROIDTC="${ANDROID_TC_ARM:-$ANDROID_TC/arm}"
-export TCBINARIES="$ANDROIDTC/bin"
+export TOOLCHAINROOT="$NDK_ROOT/toolchains/llvm/prebuilt/linux-x86_64/"
+export TCBINARIES="$TOOLCHAINROOT/bin"
 export TCINCLUDES="$ANDROIDTC/arm-linux-androideabi"
-export TCSYSROOT="$ANDROIDTC/sysroot"
-export STDCPPTC="$TCINCLUDES/lib/armv7-a/libstdc++.a"
+export TCSYSROOT="$TOOLCHAINROOT/sysroot"
```

Comment:
Before this change `TCSYSROOT` target was folder with cache. After this change everything will be installed outside of cache directory

Reply:
Before the change TCINCLUDES is the target inside the cache folder.
After the change, also TCINCLUDES is the target inside the cache folder.

Reply:
I'm talking here about `TCSYSROOT`

Reply:
TCSYSROOT is not a target, never was.

Reply:
Everything in this change install inside the cache folder, in separate folder for each platform according to `TCINCLUDES ` like before this change.

Reply:
Did you check this? I'm pretty much sure that there will be nothing inside cache directory after `make install`

Reply:
I will check it again.

Reply:
run travis_build_all.sh in my wsl2 after delete cache folder:
![travis_build_all](https://user-images.githubusercontent.com/7043539/86477145-8420d900-bd50-11ea-8268-dfd632700bf9.PNG)

Here is a result from openssl arm just finish:
![libcrypto](https://user-images.githubusercontent.com/7043539/86477217-a581c500-bd50-11ea-9cfc-8e3134cbd0e9.PNG)


I can wait and show you all the content of the folder when it finish if you want to.